app/src/main/java/com/github/epfl/meili/posts/forum/ForumActivity.kt

```diff
@@ -70,6 +66,7 @@ class ForumActivity : MenuActivity(R.menu.nav_forum_menu), AdapterView.OnItemSel
 
     private lateinit var poi: PointOfInterest
 
+    override fun getActivity(): AppCompatActivity = this
```

Comment:
One small question, is this function really necessary? Seems redundant to have a function that called on an object that returns the object itself. But maybe I am not understanding the use of it. Could you clarify, please?

Reply:
This function is used in the interface `PostListActivity` in functions that need the activity. The other option would be to pass the `this` to the interface's functions when they are called from the activity, but I found this to be the better option. What do you think ?

Reply:
Awesome! Makes sense! Thanks!